plugins/org.eclipse.n4js/src/org/eclipse/n4js/typesystem/n4js.xsemantics

```diff
@@ -695,9 +698,10 @@ from {
 			val staticAccess = (targetTypeRef instanceof TypeTypeRef)
 			val checkVisibility = false // access modifiers checked in validation
 			val scope = memberScopingHelper.createMemberScope(targetTypeRef, expr, checkVisibility, staticAccess)
-			val member = memberScopingHelper.findUniqueMemberForName(scope, memberName, staticAccess)
-			if(member !== null) {
-				G |- member : var TypeRef memberTypeRef
+			val memberDesc = scope.getSingleElement(qualifiedNameConverter.toQualifiedName(memberName));
```

Comment:
The original method `findUniqueMemberForName` filters `IEObjectDescriptionWithError`. In this refactored version, shouldn't we check `! (memberDesc instanceof IEObjectDescriptionWithError)` as well?